Products/Zuul/facades/__init__.py

```diff
@@ -155,12 +155,17 @@ def findMatchingOrganizers(self, organizerClass, organizerPath, userFilter):
         if self.validRegex(filterRegex):
             orgquery = (Eq('objectImplements','Products.ZenModel.%s.%s' % (organizerClass, organizerClass)) &
                         MatchRegexp('uid', filterRegex))
-            paths = [ "{0}/*".format(b.getPath()) for b in IModelCatalogTool(self._dmd).search(query=orgquery)]
+            paths = [ "{0}/".format(b.getPath()) for b in IModelCatalogTool(self._dmd).search(query=orgquery)]
```

Comment:
Do we need the trailing slash? right now it does not matter bc path is tokenized, but we may change it in the future?